extensions/_example/src/index.js

```diff
@@ -12,8 +12,9 @@ export default {
    */
 
   preRegistration({
-    servicesManager,
-    configuration: extensionConfiguration,
+    servicesManager = {},
+    commandsManager = {},
```

Comment:
Why would `servicesManager` or `commandsManager` ever be undefined?

Reply:
Updated.